src/app/components/evolution/evolution.controller.js

```diff
@@ -13,13 +13,22 @@ angular.module('optimizer.evolution.controller', [])
         }, 0);
       };
 
-      $scope.getEvolutions = function (candy) {
+      $scope.getEvolutions = function (candy, transfer=false) {
```

Comment:
Ah, you caught me! The guy who does not do javascripts for a living. (Or _ever_.)

I was thinking of Python again, but when I tested it, nothing bad happened, and here we all are.

@justinleewells I can re-spin the PR, or, because I am a lowly C developer, you may want to touch up the frontend ramifications of my change _anyway_. I was too timid to just /replace/ the evolutions column, and as a result the Pokemon name column got a little crowded, but I wasn't sure of the best way to fix it.

@therealklanni Thank you kindly for the review.

--js